### PR TITLE
Anon opt-in - first pass

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by adamsilver** [comment]
:+1:

File: test/app.spec.js [hunk 1]
```diff
@@ -175,6 +175,44 @@ describe('Next Opt In Out', function() {
...
+				request(app)
+					.get('/__anon-opt-in?location=/home/uk')
+					.set('FT-Site', 'FALCON')
+					.expect('Location', /www\.ft\.com/, done);
+			});
+
+			it('Should not opt-in people with FTSession', function(done){
+				request(app)
+					.get('/__anon-opt-in?location=/home/uk')
+					.set('FT-Ssession-Token', 'skdcdsjbdj')
```
Comment (keirog) [inline test/app.spec.js]: Ssession = Session

Comment (adamsilver) [inline test/app.spec.js]: Is the 'expect' not working as expected if this typo never caused failure?

**Comment by keirog** [comment]
What's the difference between [X-FT-Session](https://github.com/search?utf8=%E2%9C%93&q=user%3AFinancial-Times+X-FT-Session&type=Code&ref=searchresults), [X-FT-Session-Token](https://github.com/search?utf8=%E2%9C%93&q=user%3AFinancial-Times+X-FT-Session-Token&type=Code&ref=searchresults) and [FT-Session-Token](https://github.com/search?utf8=%E2%9C%93&q=user%3AFinancial-Times+FT-Session-Token&type=Code&ref=searchresults)?

**Comment by aintgoin2goa** [comment]
anything beginning with an X is deprecated as per http spec: https://tools.ietf.org/html/rfc6648.

**Pull request merged**